requirements.txt

```diff
@@ -1,3 +1,3 @@
 Django >= 1.8, < 1.9
-pymongo==2.4.1
+pymongo>=2.7.2,<4.0.0
```

Comment:
I'm now confused how event-tracking could have been installed in edx-platform, since it is depending upon 2.9.1. It appears to be installed here:

https://github.com/edx/edx-platform/blob/aa3d6e87913e1e2e736070da6a569a16ee5044b0/requirements/edx/github.txt#L76